modules/oe_webtools_analytics/tests/src/Functional/ConfigurationTest.php

```diff
@@ -32,13 +31,16 @@ class ConfigurationTest extends BrowserTestBase {
    * Tests if the configuration for the Webtools library is present on the page.
    */
   public function testLibraryLoading(): void {
-    $config = \Drupal::configFactory()
-      ->getEditable(AnalyticsEventInterface::CONFIG_NAME)
-      ->set('siteID', '123')
-      ->set('sitePath', 'ec.europa.eu')
-      ->set('instance', 'testing');
-    $config->save();
+    $user = $this->createUser(['administer webtools analytics']);
+    $this->drupalLogin($user);
+    $this->drupalGet('admin/config/system/oe_webtools_analytics');
+    $page = $this->getSession()->getPage();
+    $page->fillField('Site ID', '123');
+    $page->fillField('Site path', 'ec.europa.eu');
+    $page->fillField('Instance', 'testing');
+    $page->pressButton('Save configuration');
```

Comment:
Why are we changing from direct modifying config to update of config through UI?
I thought it was fine as before. Anyway, we are changing it through UI later in this test.

Reply:
Just making sure the field can be filled in both with digits only and alphanumeric values.